clients/unshaded/src/main/java/tachyon/client/lineage/AbstractLineageClient.java

```diff
@@ -91,4 +93,19 @@ public boolean deleteLineage(long lineageId, DeleteLineageOptions options) throw
       mContext.releaseMasterClient(masterClient);
     }
   }
+
+  /**
+   * Transfors the list of @{link TachyonURI} in a new list of Strings,
+   * where each string is @{link TachyonURI#getPath()}
+   * @param uris
+   * @return
+   */
+  private List<String> stripURIList(List<TachyonURI> uris) {
+    return Lists.transform(uris, new Function<TachyonURI, String>() {
+      @Override
+      public String apply(TachyonURI input) {
+        return input.getPath();
+      }
+    });
```

Comment:
I don't think we use this pattern anywhere else in the codebase. Perhaps it would be better for code style if we wait till java 8 (where its much nicer)?